Example/Tests/Specs/PinViewSpec.swift

```diff
@@ -0,0 +1,180 @@
+//
+//  PinViewSpec.swift
+//  Delirium
+//
+//  Created by Jelle Vandebeeck on 14/10/2016.
+//  Copyright © 2016 CocoaPods. All rights reserved.
+//
+
+import Quick
+import Nimble
+import Nimble_Snapshots
+import Delirium
+
+class PinViewSpec: QuickSnapshotSpec {
+
+    override func spec() {
+        describe("pin view") {
+            var view: PinView!
+            beforeEach{
+                let bundle = Bundle(for: self.classForCoder)
+                let storyboard = UIStoryboard(name: "PinSpec", bundle: bundle)
+                let controller = storyboard.instantiateInitialViewController()!
+                view = controller.view.subviews.first as? PinView
```

Comment:
❓Kan je niet gewoon een view instantieren. Moet het in een viewController? Of wordt hij dan niet getekend?